demos/projects/ST/stm32h745i-disco/cm7/config/demo_config.h

```diff
@@ -7,6 +7,12 @@
 /* FreeRTOS config include. */
 #include "FreeRTOSConfig.h"
 
+/*
+ * This Model ID is tightly tied to the code implementation in `sample_azure_iot_pnp_simulated_device.c`
+ * If you intent to test a different Model ID, please provide the implementation of the model on your application.
```

Comment:
add link to the model json that is in github

Reply:
That's a good idea.

Reply:
Updated in all demo_config.h files that use this model.